# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mp4parse_capi/tests/test_rotation.rs
# Commit: a651d4323066
# Full Hash: a651d43230664837c308663382ccc3bcd7298dc2
# Author: zaggy1024 <Zaggy1024@gmail.com>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1788119
# File Overlap Count: 1
# Description:
#   Bug 1788119 - Part 4 - Update mp4parse-rust for AVIS support. r=kinetik,glandium,supply-chain-reviewers
#   
#   Depends on D156652
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156653
# ==============================================================================

diff -r be008ddb2418 -r a651d4323066 third_party/rust/mp4parse_capi/tests/test_rotation.rs
--- a/third_party/rust/mp4parse_capi/tests/test_rotation.rs	Fri Jan 27 21:25:50 2023 +0000
+++ b/third_party/rust/mp4parse_capi/tests/test_rotation.rs	Fri Jan 27 21:25:51 2023 +0000
@@ -1,4 +1,3 @@
-extern crate mp4parse_capi;
 use mp4parse_capi::*;
 use std::io::Read;
 